docs/prow/kyma-release-process.md

```diff
@@ -24,90 +24,54 @@ The internal release process is documented as a comment in [this](https://github
 7. Attach artifacts and a `README.md` document to the release on GitHub.
 
 ## Release process in Prow
+When using standard kyma buildpacks, like buildpack for Go or Node.js applications, a job for a component additionally clones `test-infra` repository to access build scripts.
+To gain reproducibility, we have to define separate release jobs for every component for every release. In the release job, cloning of `test-infra` repository should be done from equivalent release branch, 
+i.e. when defining the job for releasing version 0.6, test-infra should be cloned also from branch `release-0.6`.
 
-The proposed release process looks as follows in Prow:
+Considering that Presubmit jobs are more powerful than Postsubmit jobs, all activities around releasing new version should be done in the context of a pull request to a release branch.
+When creating PR, job for every component will be triggered. 
+After introducing all necessary changes, like modifying of `values.yaml` with a new version of components, release master 
```

Comment:
There will be 2 PR:
- one in test-infra repository, where you modify RELEASE_VERSION file
- one in kyma, where you modify values.yaml

We will have kyma-integration job for releases, it will be triggered by a comment on PR.

Reply:
Ok, now it's clear to me.